docs/user-guide/dream/dream-advanced-powder-reduction.ipynb

```diff
@@ -0,0 +1,480 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9",
+   "metadata": {},
+   "source": [
+    "## Alternative run normalizations\n",
+    "\n",
+    "The [basic example](./dream-data-reduction.rst) normalizes the detector data by a monitor that was histogrammed in wavelength.\n",
+    "ESSdiffraction provides some alternatives.\n",
+    "\n",
+    "### Normalize by integrated monitor\n",
+    "\n",
+    "Instead of computing a histogram of the monitor data, we can integrate over all bins to get a single intensity value for the monitor.\n",
+    "To do so, specify `ess.powder.RunNormalization.monitor_integrated` when constructing the workflow.\n",
+    "This will insert [normalize_by_monitor_integrated](../../generated/modules/ess.powder.correction.normalize_by_monitor_integrated.rst) into the workflow."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "10",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workflow = dream.DreamGeant4Workflow(run_norm=powder.RunNormalization.monitor_integrated)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11",
+   "metadata": {},
+   "source": [
+    "Then set all parameters as before:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workflow[Filename[SampleRun]] = dream.data.simulated_diamond_sample()\n",
+    "workflow[Filename[VanadiumRun]] = dream.data.simulated_vanadium_sample()\n",
+    "workflow[Filename[BackgroundRun]] = dream.data.simulated_empty_can()\n",
+    "workflow[CalibrationFilename] = None\n",
+    "\n",
+    "workflow[MonitorFilename[SampleRun]] = dream.data.simulated_monitor_diamond_sample()\n",
+    "workflow[MonitorFilename[VanadiumRun]] = dream.data.simulated_monitor_vanadium_sample()\n",
+    "workflow[MonitorFilename[BackgroundRun]] = dream.data.simulated_monitor_empty_can()\n",
+    "workflow[CaveMonitorPosition] = sc.vector([0.0, 0.0, -4220.0], unit=\"mm\")\n",
+    "\n",
+    "workflow[dream.InstrumentConfiguration] = dream.InstrumentConfiguration.high_flux\n",
+    "# Select a detector bank:\n",
+    "workflow[NeXusDetectorName] = \"mantle\"\n",
+    "# We drop uncertainties where they would otherwise lead to correlations:\n",
+    "workflow[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.drop\n",
+    "# Edges for binning in d-spacing:\n",
+    "workflow[DspacingBins] = sc.linspace(\"dspacing\", 0.0, 2.3434, 201, unit=\"angstrom\")\n",
+    "\n",
+    "# Do not mask any pixels / voxels:\n",
+    "workflow = powder.with_pixel_mask_filenames(workflow, [])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "13",
+   "metadata": {},
+   "source": [
+    "And compute the result:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = workflow.compute(IofTof)\n",
+    "result.hist().plot()"
```

Comment:
This plot is empty in the generated docs.

Reply:
Not completely empty for me but definitely broken:
![integrated](https://github.com/user-attachments/assets/2400cc70-a377-4b0b-bb6a-03893d3e4815)

This seems to be a problem with the integrated monitor norm. I don't know what. But iit should not have been caused by this PR.

Reply:
We should really have a test in place that catches this...

Reply:
The plot looks fine now